src/asammdf/mdf.py

```diff
@@ -176,26 +176,26 @@ class MDF:
 
         .. versionchanged:: 6.3.0 make the default None
 
-    use_display_names (\*\*kwargs) : bool
+    use_display_names (**kwargs) : bool
         keyword only argument: for MDF4 files parse the XML channel comment to
         search for the display name; XML parsing is quite expensive so setting
         this to *False* can decrease the loading times very much; default
         *True*
-    remove_source_from_channel_names (\*\*kwargs) : bool
-        remove source from channel names ("Speed\XCP3" -> "Speed")
-    copy_on_get (\*\*kwargs) : bool
+    remove_source_from_channel_names (**kwargs) : bool
+        remove source from channel names ("SpeedXCP3" -> "Speed")
```

Comment:
Here we have to keep the slash because i's the way some applications were embedding the source name in the channel name